internal/backends/sqlite/metadata/metadata.go

```diff
@@ -32,7 +33,7 @@ import (
 
 const (
 	// DefaultColumn is a column name for all fields.
-	DefaultColumn = "_ferretdb_sjson"
+	DefaultColumn = backends.ReservedPrefix + "_sjson"
```

Comment:
Some tests start failing due to this test. `DefaultColumn` used to be `_ferretdb_sjson`, but now it's `_ferretdb__sjson`. Should it remain `_ferretdb_sjson` or do we use double `_` on purpose (if so, tests need to be updated)?

Reply:
P.S. I guess it's not on purpose as this change breaks back compatibility. Fun fact: integration tests would pass. That's why I avoid using constants in unit tests: sometimes constant-less tests help catch such things!